backend/test/card_game/api_messages_test.clj

```diff
@@ -0,0 +1,63 @@
+(ns card-game.api-messages-test
+  (:require [expectations.clojure.test :refer :all]
+            [card-game.api.base :as api]
+            [configs :as configs]))
+
+(defexpect status-check
+  ; Game tracks status correctly
+  (expect
+    configs/no-opp
+    (:status (api/create-game)))
+  (expect
+    configs/play
+    (-> (api/create-game)
+        :game-id
+        (api/add-player)
+        :status))
+  (expect
+    configs/no-opp
+    (let [game (api/create-game)]
+      (:status (api/get-game (:game-id game) (:player-id game)))))
+  (expect
+    configs/wait
+    (let [game (api/create-game)]
```

Comment:
I think this test can be more readable by adding `game-id (:game-id game)` to the `let` and changing `(->` syntax for a `(do` syntax.